packages/docgen/test/get-type-annotation.js

```diff
@@ -19,7 +22,7 @@ describe( 'Type annotations', () => {
 			type: 'number',
 		};
 		const node = {};
-		const result = getTypeAnnotation( tag, node );
+		const result = getTypeAnnotation( tag, node, 0 );
```

Comment:
What do you think about making the default param for index be 0? (Or maybe it makes more sense to just be explicit, idk)